### ISSUE TITLE
Markers show on wrong world

### ISSUE THREAD

**Issue by pordini** [OpenGIS/Waymark#35]
Markers are shown in the "wrong" world, as here: 
![image](https://github.com/OpenGIS/Waymark/assets/12774050/8ef6bacf-e66e-4e27-9934-34713864e5a0)

It seems like the map goes on and on, you can scroll and the markers never show up on any of the duplicate worlds. Most map applications show all the map info on all the worlds, I think that makes more sense.

**Comment by pordini** [comment]
(it is possible to resolve the placement issue by moving the yellow markers to the right, in the example, but that only addresses this particular situation and not the underlying issue, I think)

**Comment by morehawes** [comment]
Hi @pordini,

Thank you for reaching out! It took me a little while to find, but someone else [reported something similar](https://wordpress.org/support/topic/display-problem-collections/) a little while ago. They too were having issues with multiple worlds and I was unable to get to the bottom of it then :-/

I will try to dig into this when I get more time. If possible, could you provide the steps necessary to replicate this issue in Waymark from a blank Map?

Cheers,

Joe

**Comment by pordini** [comment]
Ah, I had searched and didn't find it, sorry. 

The steps to recreate it:
Create new map
Add marker wherever (Let's say in Paris, France)
Scroll the map left until you are at Paris again
Add another marker in Paris
Publish

You now have two markers, one in Paris and on in the other Paris.

**Comment by pordini** [comment]
You can keep doing this btw.

![image](https://github.com/OpenGIS/Waymark/assets/12774050/06162905-0c4f-4485-8ee8-61afb5f4a898)

**Comment by morehawes** [comment]
> Ah, I had searched and didn't find it, sorry.

Not at all! The WP.org support search is terrible and it even took me some effort! :0)

> The steps to recreate it:

Thank you, I will try to sit down with this soon and see if I can figure out a way to improve this.
